pkgs/by-name/dm/dmenu-rs/package.nix

```
@@ -0,0 +1,109 @@
+{
+  aspell,
+  cargo,
+  expat,
+  fetchFromGitHub,
+  fontconfig,
+  lib,
+  libXft,
+  libXinerama,
+  m4,
+  pkg-config,
+  python3,
+  rustPlatform,
+  stdenv,
+  xclip,
+  xdg-utils,
+  enablePlugins ? false,
+}:
```

```suggestion
{
  lib,
  stdenv,
  fetchFromGitHub,

  # nativeBuildInputs
  cargo,
  m4,
  pkg-config,
  python3,
  rustPlatform,

  # buildInputs
  expat,
  fontconfig,
  libXft,
  libXinerama,
  aspell,
  xclip,
  xdg-utils,

  enablePlugins ? false,
}:
```
Arguments should be sorted in the appearing order. Also, `lib` and `stdenv` should go at the top.

---

I personally prefer when the sections are separated, but you are free to ignore this suggestion. This is more subjective.

---

> Do you prefer the sections to have a comment (e.g., `nativeBuildInputs`) as well?

I tend to do so when I package things. Again, this is quite subjective.